app/src/main.js

```diff
@@ -16,6 +16,13 @@ electronDownload();
 const APP_ARGS_FILE_PATH = path.join(__dirname, '..', 'nativefier.json');
 const appArgs = JSON.parse(fs.readFileSync(APP_ARGS_FILE_PATH, 'utf8'));
 
+if (appArgs.processEnvs) {
+  Object.keys(appArgs.processEnvs).forEach((key) => {
+    // eslint-disable-next-line no-undef
```

Comment:
Should I put it in the `.eslintrc.yml` file or just inline it (`/* eslint-env node */`)?